_config.yml

```
@@ -101,8 +101,8 @@ social:
 # Home Assistant release details
 current_major_version: 0
 current_minor_version: 109
-current_patch_version: 1
-date_released: 2020-04-30
+current_patch_version: 2
+date_released: 2020-04-31
```

The month of April only has 30 days :wink: 

```suggestion
date_released: 2020-04-30
```

---

Sharp! UTC already passed into the first of May, corrected it 👍